app/models/service_template_transformation_plan_task.rb

```diff
@@ -52,13 +52,21 @@ def task_active
   # This method returns true if all mappings are ok. It also preload
   #  virtv2v_disks and network_mappings in task options
   def preflight_check
+    raise 'OSP destination and source power_state is off' if destination_ems.emstype == 'openstack' && source.power_state == 'off'
+    #####
+    # this block is the essence of automate method: assesstransformation and its effect
```

Comment:
Comment is "Basmati Rice"

Reply:
I am so aware of this!!! And you have no idea how many times I re-edit this thing 😭 

Intended to remove this after the review!

Reply:
If the comment is just for the PR reviewers you can actually put a github comment on the line instead of adding a comment to the source code that will later be deleted.  (appreciate helping the reviewers though haha)